docs/advanced/releases.md

```diff
@@ -0,0 +1,104 @@
+
+If this is a **Release Candidate version** release, the steps to follow are:
+
+1. Go to the release branch `git checkout release/x.y.z`.
+1. Update `.decidim-version` to the new version `x.y.z.rc1`
+1. Run `bin/rake update_versions`, this will update all references to the new version.
+1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
+1. Run `bin/rake webpack`, this will update the JavaScript bundle.
+1. Commit all the changes: `git add . && git commit -m "Bump to rcXX version" && git push origin release/x.y.z`.
+1. Follow the link resulting from the previous command to create the PR for the new version.
+1. Usually, at this point, a deploy to meta-decidim is done during, at least, one week to validate the stability of the version.
+
+### During the validation period
+
+1. During the validation period bugfixes will go directly to the current `release/x.y.z` branch and ported to `develop`.
+1. During the validation period, translations to the officially supported languages must be added to Crowdin and when done, merged into `release/x.y.z`.
+
+### Major/Minor versions
+
+Once Release Candidates have been tested, merging into `master` after the validation period can be done.
+
+1. Go to the release branch `git checkout release/x.y.z`.
+1. Update `.decidim-version` by removing the `.rcN` suffix, leaving a clean version number like `x.y.z`
+1. Run `bin/rake update_versions`, this will update all references to the new version.
+1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
+1. Run `bin/rake webpack`, this will update the JavaScript bundle.
+1. Commit all the changes: `git add . && git commit -m "Bump to v0.XX.0 final version" && git push origin release/x.y.z`.
+1. Follow the link resulting from the previous command to create the PR for the new version, but still don't merge it. The base for the PR should be `master`.
+1. Before merging the PR to upgrade `master`, create the stable branch for the previous vesion. So, if this is the release of v0.22.0, branch `0.21-stable` from `master`. These stable branches will be able to receive bugfixes, backports and will be the origin of patch releases for older releases.
+1. Merge the PR to `master` and remove `release/x.y.z` branch.
+1. Run `git checkout master && bin/rake release_all`, this will create all the tags, push the commits and tags and release the gems to RubyGems.
+1. Once all the gems are published you should create a new release at this repository, just go to the [releases page](https://github.com/decidim/decidim/releases) and create a new one.
+1. Update Decidim's Docker repository as explained in the Docker images section.
+1. Update Crowdin synchronization configuration with Github:
+    1. Add the new `x.y-stable` branch.
+    1. Remove from Crowdin branches that are not officially supported anymore. That way they don't synchronize with Github.
+
```

Comment:
I would add here instruction to generate the next stable branch, this way is already created and people can saftley use `git: "https://github.com/decidim/decidim", branch: "0.XX-stable"` in their Gemfiles without waiting for the first patch.